runtime/hsa-runtime/core/driver/xdna/uapi/amdxdna_accel.h

```diff
@@ -6,7 +6,7 @@
 #ifndef AMDXDNA_ACCEL_H_
```

Comment:
Did we ever figure out a way to detect if the driver we compiled with and the installed driver differ?